monkey/agent_plugins/exploiters/rdp/src/rdp_credentials_generator.py

```diff
@@ -9,35 +9,55 @@
 
 
 def generate_rdp_credentials(
-    credentials: Sequence[Credentials], domains: Sequence[str]
+    credentials: Sequence[Credentials], domains: Sequence[str], running_from_windows: bool
```

Comment:
"from" or "on"? Let's pick one.
```suggestion
    credentials: Sequence[Credentials], domains: Sequence[str], running_on_windows: bool
```

Reply:
I think we have "from" in Powershell. The purpose is the same so I say lets go with consistency.

Reply:
Then it needs to be fixed in `plugin.py`, which has `running_on_windows`.